client/src/app/shared/forms/form-validators/video-validators.service.ts

```diff
@@ -79,7 +79,7 @@ export class VideoValidatorsService {
     }
 
     this.VIDEO_SUPPORT = {
-      VALIDATORS: [ Validators.minLength(3), Validators.maxLength(500) ],
+      VALIDATORS: [ Validators.minLength(3), Validators.maxLength(1000) ],
       MESSAGES: {
         'minlength': this.i18n('Video support must be at least 3 characters long.'),
         'maxlength': this.i18n('Video support cannot be more than 500 characters long.')
```

Comment:
```suggestion
         'maxlength': this.i18n('Video support cannot be more than 1000 characters long.')
```

Reply:
I've made the fix, not sure where my focus went haha